src/ServiceManagement/RemoteApp/Commands.RemoteApp/packages.config

```diff
@@ -0,0 +1,26 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Hyak.Common" version="1.0.2" targetFramework="net45" />
+  <package id="Microsoft.Azure.Common" version="2.0.2" targetFramework="net45" />
+  <package id="Microsoft.Azure.Common.Authentication" version="1.0.10-preview" targetFramework="net45" />
```

Comment:
This needs to be updated to match the Azure.Common.Authentication version used in all the other cmdlets